Libs/MRML/Core/vtkMRMLStorageNode.cxx

```diff
@@ -46,6 +46,7 @@ vtkMRMLStorageNode::vtkMRMLStorageNode()
   this->URI = NULL;
   this->URIHandler = NULL;
   this->UseCompression = 1;
+  this->CompressionLevel = 1;
```

Comment:
Need to add CompressionLevel to read/write xml and print methods.

I'm also wondering if we should use DEFAULT, NONE, LOW, MEDIUM, HIGH enums for compression levels. If we ever expose this to users, it would be nicer to show fewer, more explicit options than 0-9. We would also have more flexibility in how we map the enums to actual compression levels in various storage nodes.

We should at least have a default compression level so that we can easily re-map default compression levels and don't litter storage nodes with explicit compression level values. If we keep compression level as int then we could use -1 as "default" level (which would not be written to XML file and would be translated to actual compression level dynamically, currently to 1).

Reply:
Ok, I added print and XML methods, an enum, and -1 <-> default mapping.